prettier/format/src/index.ts

```diff
@@ -23,7 +24,7 @@ workerMain(async (a) => {
       : (await resolveConfig(args.config, { config: args.config })) ||
         undefined;
   if (options?.plugins) {
-    const contextUrl = pathToFileURL(dirname(args.config!));
+    const contextUrl = pathToFileURL(path.dirname(args.config!));
```

Comment:
As it stands better_rules_javascript does not support Prettier's per-filetype `overrides` because we only call Prettier's `resolveConfig` once (3 lines above here). Also, Prettier would require an extra disk read per file being formatted anyway.

I put in `fileExtOverrides` below as a lightweight alternative for Rivet's needs. We need an override to turn off trailing commas in Angular templates because Prettier 3+ always puts trailing commas in .html files - it assumes Angular 16+. We're on Angular 14, which prohibits trailing commas.